src/Driver.php

```diff
@@ -70,7 +70,7 @@ public static function getUserAgent()
      */
     public function __construct($uri, Configuration $configuration = null)
```

Comment:
This could be changed to `BaseConfiguration`. That would give the user more flexibility and we would not have any type error from the `GraphDatabaseInterface`.